package.json

```diff
@@ -42,7 +42,7 @@
     "start": "react-app-rewired start",
     "dev": "DISABLE_ESLINT_PLUGIN=true TSC_COMPILE_ON_ERROR=true REACT_APP_BACKEND=http://localhost:8765 npm start",
     "build": "react-app-rewired build",
-    "build:embedded": "rm -rf build && PUBLIC_URL=/monitoring/ REACT_APP_BACKEND=http://localhost:8765 npm run build",
+    "build:embedded": "rm -rf build && PUBLIC_URL=. REACT_APP_BACKEND=http://localhost:8765 npm run build",
```

Comment:
I really want to add a comment to this setting, but json file wouldn't allow this. Will find a way to clarify why is it set up this way.